### ISSUE TITLE
Does the color attribute work?

### ISSUE THREAD

**Issue by ytalp17** [snehilvj/dash-iconify#8]
May I learn whether the color attribute work as expected? If yes, could you please provide me a way to use it.

**Comment by AnnMarieW** [comment]
Yes, it works, there are multiple examples in the docs.  Can you give a complete minimal example of it not working?

**Comment by ytalp17** [comment]
from dash import Dash, html
from dash_iconify import DashIconify

def get_icon(icon, height, color=None):
    return DashIconify(icon=icon, height=height, color=color)

app = Dash(__name__)

app.layout = html.Div([
    html.Div([
        "Icon:",
        get_icon("emojione-monotone:letter-s", 18, color="red")  # working example
    ], style={'display': 'flex', 'alignItems': 'center'})
])

if __name__ == '__main__':
    app.run_server(debug=True)
    
I just happen to see that color attribute does not work for certain icons (e.g. twemoji:letter-s).
Is there any way to change color of the icon on my example for instance? If no, is there any way to tell which icons have changable color?

**Comment by AnnMarieW** [comment]
You can find more information in the Iconify docs  https://iconify.design/docs/icons/icon-basics.html#colors
